### PR TITLE
Fixes #9635 Fixes #7023

### LINKED ISSUE #7023: Grills shocking even when covered with a window
(WEB REPORT BY: golfer45 REMOTE: 158.69.55.2:7777)
To reporduce this, break one of the grills above the bridge on box, walk to where the grill used to be. Now walk into the grills next to it, it will shock you even while covered in glass

### LINKED ISSUE #9635: Fulltile windows don't prevent you from bumping into stuff, notably, shocked grilles
To reproduce: Build shocked grille, then build fulltile window on top, marvel as you're shocked when you bump into that square.
Alternatively: Go to any of the many windows in the station that have shocked grilles under border windows under fulltile windows, break the border windows first (i.e. by throwing shit at it), walk into that square, be shocked.

### PR DESCRIPTION
**Pull request by ghost**
Fixes #9635 Fixes #7023

### CODE REVIEW

**Comment by Intigracy** [comment]
7023 too

**Comment by Exxion** [comment]
The title doesn't work for autoclosing. It's stupid but true.
Only the OP body and commit messages

**Comment by ghost** [comment]
FFS is it doing what I want yet?

**Comment by Exxion** [comment]
Yes, that should work

**Comment by gbasood** [comment]
It does actually work with auto closing

File: code/game/objects/structures/fullwindow.dm [hunk 1]
```diff
@@ -12,12 +12,15 @@
...
+	penetration_dampening = 1
+
+
 	cracked_base = "fcrack"
 
 /obj/structure/window/full/New(loc)
 
 	..(loc)
-	flags &= ~ON_BORDER
+	flags |= ON_BORDER
```
Comment (clusterfack) [inline code/game/objects/structures/fullwindow.dm]: I believe you can remove this line entirely, it should already have the flag

Comment (gbasood) [inline code/game/objects/structures/fullwindow.dm:23]: I thought someone added this recently?

Comment (Intigracy) [inline code/game/objects/structures/fullwindow.dm:23]: This is removing the on_border flag currently when you spawn a fulltile window, isn't it?

I thought that was intended?

**Comment by clusterfack** [comment]
?

**Pull request merged**